Makefile

```diff
@@ -158,18 +162,22 @@ test\:ci:
 
 .PHONY: test\:go
 test\:go:
+	$(MAKE) -C ./tests/lenses build
```

Comment:
todo: remove according to discussion on discord.

Reply:
Already included in the description-level task